### PR TITLE
trigger workflow from branch

### PR DESCRIPTION
**Pull request by cmentzer**

### CODE REVIEW

File: .github/workflows/scryfall_bulkdata.yml [hunk 1]
```diff
@@ -2,10 +2,11 @@ name: Scryfall Bulkdata Validation
 
 on:
   push:
-    branches: [dev]
+    branches: [dev, cm/scryfall-workflow-fix]
+
   # Every Monday at 07:00 UTC
-  schedule:
-    - cron: "0 7 * * 1"
```
Comment (iambroadband) [inline .github/workflows/scryfall_bulkdata.yml:8]: probably want to change this back before merging

File: .github/workflows/scryfall_bulkdata.yml [hunk 2]
```diff
@@ -2,10 +2,11 @@ name: Scryfall Bulkdata Validation
 
 on:
   push:
-    branches: [dev]
```
Comment (iambroadband) [inline .github/workflows/scryfall_bulkdata.yml:5]: also this

File: .github/workflows/scryfall_bulkdata.yml [hunk 3]
```diff
@@ -49,12 +50,13 @@ jobs:
 
       - name: Load Scryfall bulkdata
         run: |
-          source .venv/bin/activate
           echo "docker compose up -d"
           docker compose up -d
           echo "scooze load cards --all --force-download --concise"
+          source .venv/bin/activate
```
Comment (iambroadband) [inline .github/workflows/scryfall_bulkdata.yml:56]: is this an order matters type of thing?

**Comment by iambroadband** [comment]
LGTM aside from the 2 comments

File: .github/workflows/scryfall_bulkdata.yml (hunk 3 above)
Comment (cmentzer) [reply .github/workflows/scryfall_bulkdata.yml:56]: it shouldn't. But I like it down there better.

**Review by iambroadband — APPROVED** [review]

**Pull request merged**